python/docs/source/development/ps_design.rst

```diff
@@ -4,40 +4,40 @@ Design Principles
 
 .. currentmodule:: pyspark.pandas
 
-This section outlines design principles guiding the Koalas project.
+This section outlines design principles guiding the pandas APIs on Spark.
 
 Be Pythonic
 -----------
 
-Koalas targets Python data scientists. We want to stick to the convention that users are already familiar with as much as possible. Here are some examples:
+Pandas APIs on Spark targets Python data scientists. We want to stick to the convention that users are already familiar with as much as possible. Here are some examples:
 
-- Function names and parameters use snake_case, rather than CamelCase. This is different from PySpark's design. For example, Koalas has `to_pandas()`, whereas PySpark has `toPandas()` for converting a DataFrame into a pandas DataFrame. In limited cases, to maintain compatibility with Spark, we also provide Spark's variant as an alias.
+- Function names and parameters use snake_case, rather than CamelCase. This is different from PySpark's design. For example, pandas APIs on Spark has `to_pandas()`, whereas PySpark has `toPandas()` for converting a DataFrame into a pandas DataFrame. In limited cases, to maintain compatibility with Spark, we also provide Spark's variant as an alias.
```

Comment:
```suggestion
- Function names and parameters use snake_case, rather than CamelCase. This is different from PySpark's design. For example, pandas APIs on Spark have `to_pandas()`, whereas PySpark has `toPandas()` for converting a DataFrame into a pandas DataFrame. In limited cases, to maintain compatibility with Spark, we also provide Spark's variant as an alias.
```